### PR TITLE
add portfolio project contribution guide in readme

### PR DESCRIPTION
**Pull request by saadamir3180**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -428,6 +429,141 @@ cards: [
...
+   - [10. Add Brief Description](#10-add-a-brief-description)
+   - [11. Add Client's Challenge](#11-add-the-challenge-client-faced)
+   - [12. Add Final Results](#12-add-final-results)
+3. [Final Example Portfolio Object](#final-example-portfolio-object)
+
+---
+
+## Steps for Adding a New Portfolio Project
+
+You need to modify the `VAPTPortfolio` or `PDPortfolio` list in `data/portfolio.js` and add the final object to the respective list to upload a new portfolio project.
```
Comment (njmulsqb) [inline README.md:460]: What if we need to add a third category of portfolio (and more)?

File: README.md [hunk 2]
```diff
@@ -428,6 +429,141 @@ cards: [
...
+The `id` (required) is the unique identifier for a portfolio project. Adding duplicates can lead to unexpected issues in other projects. You need to assign an `id` that is one value greater than the previous project's `id` to maintain order. For example, if the previous project's `id` is `101`, use `102`, as shown below:
+```javascript
+id: 102,
+```
+
+### 2. Add a Preview Image
+The `imageSrc` (required) is used as the display picture for the project. Upload an image (dimensions: 1130x1420) to the respective folder based on the project type:  
+
+- For type `pd` (Product Design): `public/assets/img/portfolio/portfolio-pd`  
+- For type `vapt` (Penetration Testing & Vulnerability Assessment): `public/assets/img/portfolio/portfolio-pavt`
```
Comment (njmulsqb) [inline README.md:472]: its vapt not pavt

File: README.md (hunk 1 above)
Comment (saadamir3180) [reply README.md:460]: If we want to add a third category, we will likely need a developer due to the way routes and portfolio items are currently handled. 

1. **Data Structure:**  
   In `data/portfolio.js`, we have two separate arrays for the existing categories:  
   image  
   To add a new project under the third category, we will need to create a third array with a similar structure.

2. **Directory Structure:**  
   Additionally, we need to add a new directory, similar to `pd` and `vapt`, within `app/(portfolio)/our-portfolio/(projects)`:  
   image  

3. **Page Modifications:**  
   Finally, we must make the necessary modifications to the `page.jsx` and `[title]/page.jsx` files in the new directory to handle the new category appropriately.

File: README.md (hunk 2 above)
Comment (saadamir3180) [reply README.md:472]: Technically, it should be `vapt`, but there's a typo in the name of the directory: `public/assets/img/portfolio/portfolio-pavt`:
image

File: README.md (hunk 1 above)
Comment (njmulsqb) [reply README.md:460]: right, can you please add this to the readme as well.

File: README.md (hunk 2 above)
Comment (njmulsqb) [reply README.md:472]: please fix it.

**Comment by njmulsqb** [comment]
There are some commits that have been merged already in the main branch, please remove them and only include your commits. Cant merge without it.

File: README.md (hunk 1 above)
Comment (saadamir3180) [reply README.md:461]: yes

**Comment by saadamir3180** [comment]
> There are some commits that have been merged already in the main branch, please remove them and only include your commits. Cant merge without it.

Okay

**Review by njmulsqb — APPROVED** [review]

**Pull request merged**